### ISSUE TITLE
Remove reactive-lib-secrets

### ISSUE THREAD

**Issue by patriknw** [lightbend/reactive-lib#113]
I guess

**Comment by renatocaval** [comment]
probably

**Comment by eed3si9n** [comment]
Absolutely.

**Comment by TimMoore** [comment]
Is there any point in deprecating this in `reactive-lib`? Anyone migrating away from `reactive-lib` will need to stop using the API. Anyone remaining on Lagom 1.4.x and Lightbend Orchestration 1.6.x can still use it. I don't think we need to do more than write a note to the migration guide explaining how to replace use of the API with environment variables and config.

**Comment by renatocaval** [comment]
Makes sense now that we decided to not have 2.0.0.

**Comment by eed3si9n** [comment]
Discussed with Tim offline a bit more on this. One of my reasons for secrets's early exit is so we can remove it off of the docs sooner since I'd expect Orchestration 1.7 to stick around as the new way gradually picks up.

**Comment by eed3si9n** [comment]
We recently realized that a customer is using this feature, so instead of removing it completely, we are going to deprecate it in sbt-reactive-apps: https://github.com/lightbend/sbt-reactive-app/pull/172

**Comment by TimMoore** [comment]
I don't think we want to deprecate the `sbt-reactive-apps` feature, which is required to insert the secrets in to the YAML generated by `rp`. We want to deprecate the API in `reactive-lib` which is completely unnecessary (just a wrapper for accessing secrets from environment variables or the filesystem).
